core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/runtimeRoutesModels.scala

```diff
@@ -82,7 +82,7 @@ object UpdateRuntimeConfigRequest {
 
 final case class UpdateRuntimeRequest(updatedRuntimeConfig: Option[UpdateRuntimeConfigRequest],
                                       allowStop: Boolean,
-                                      updateAutopauseEnabled: Option[Boolean],
+                                      updateAutopause: Option[Boolean],
```

Comment:
is this rename intended?

Reply:
The rename was intended to help this match the corresponding variable, which is simply autopause.